web/templates/message_controls.hbs

```diff
@@ -11,7 +11,7 @@
 
     {{#unless msg/locally_echoed}}
     <div class="actions_hover message_control_button" data-tooltip-template-id="message-actions-tooltip-template" >
-        <i class="zulip-icon zulip-icon-more-vertical" role="button" aria-haspopup="true" tabindex="0" aria-label="{{t 'Message actions' }}"></i>
+        <i class="actions-button zulip-icon zulip-icon-more-vertical" role="button" aria-haspopup="true" tabindex="0" aria-label="{{t 'Message actions' }}"></i>
```

Comment:
maybe `message-actions-menu-button`? I worry in this context, "actions" sounds a bit too much like this might be the same sort of thing as `message_control_button`, which is a general class.

Reply:
Yes, definitely. That would be an improvement.